device_parser: improve error reporting

mwasilew opened the pull request:
This patch adds more verbose output to config file parsing errors.

lumag commented on device_parser.c:
```diff
@@ -22,6 +22,7 @@ static void nextsym(struct device_parser *dp)
 {
 	if (!yaml_parser_parse(&dp->parser, &dp->event)) {
 		fprintf(stderr, "device parser: error %u\n", dp->parser.error);
+		fprintf(stderr, "device parser: index %zu, line %zu, column %zu\n", dp->parser.context_mark.index, dp->parser.context_mark.line, dp->parser.context_mark.column);
```
Please wrap to some sensible width (80-100 chars)

lumag commented on device_parser.c:
```diff
@@ -43,14 +44,34 @@ int device_parser_accept(struct device_parser *dp, int type,
+		};
+
+		fprintf(stderr, "device parser: expected %s got %s\n", event_type[type], event_type[dp->event.type]);
+		fprintf(stderr, "device parser: index %zu, line %zu, column %zu\n", dp->parser.mark.index, dp->parser.mark.line, dp->parser.mark.column);
```
and  here

lumag reviewed (changes requested):
Could you please rephrase commit message to use imperative language. In other words, "This patch adds" -> "Add".

mwasilew commented:
> Could you please rephrase commit message to use imperative language. In other words, "This patch adds" -> "Add".

Fixed.

I split the long lines, but I'm not sure this is done "the right way". I didn't find any other long lines in the code to follow the style. Let me know what is the preferred way here.
